src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEmployeeInfoValidationServiceImpl.java

```diff
@@ -95,6 +103,16 @@ public String getAddressValidationMessageIfCheckPayment(String employeeId) {
             }
         }
         return validationMessage;
+    }   
+
+    @Override
+    public boolean isEmployeeGroupIdValid(String employeeGroupId) {
+        if(StringUtils.isNotBlank(employeeGroupId)){
+            Collection<String> acceptedValuesForGroupId = parameterService.getParameterValuesAsString(CUKFSConstants.ParameterNamespaces.CONCUR, 
+                    CUKFSParameterKeyConstants.ALL_COMPONENTS, ConcurParameterConstants.CONCUR_CUSTOMER_PROFILE_GROUP_ID);
+            return acceptedValuesForGroupId.stream().filter(acceptedValue -> acceptedValue.equalsIgnoreCase(employeeGroupId)).count() > 0;
```

Comment:
this depends on whether the collection of acceptedValues is empty or null. If null it will throw an exception. Currently, the implementation for the getParameterValuesAsString method returns an empty collection if no value is set so it should be fine. However, a check would be good here as that implementation could change. Will update the code.